src/app/beer_garden/local_plugins/manager.py

```diff
@@ -7,22 +7,24 @@
 import logging
 import os
 import sys
-from brewtils.models import Event, Events, System
+from brewtils.models import Event, System
 from brewtils.specification import _SYSTEM_SPEC
 from brewtils.stoppable_thread import StoppableThread
 from importlib.machinery import SourceFileLoader
 from importlib.util import module_from_spec, spec_from_file_location
-from pathlib import Path, PosixPath
+from pathlib import Path
 from random import choice
 from types import ModuleType
 from typing import Any, Dict, List, Optional
 
-import beer_garden
 import beer_garden.config as config
 from beer_garden.errors import PluginValidationError
 from beer_garden.local_plugins.env_help import expand_string
 from beer_garden.local_plugins.runner import ProcessRunner
 
+# This is ... complicated. See the PluginManager docstring
+lpm_proxy = None
+
```

Comment:
I get why you are placing this in the Plugin Manager but it is only utilized by the App. Should we be placing this here just for a global processor variable?

Reply:
It's only used in the App because this PR is just the groundwork for the real work that I'm doing. The main purpose of this refactor is so that other modules can now get access to *something* that let's the interact with the local plugin state.